--- gai-0.5.0pre6/gai.pc.in.orig	2003-09-23 20:27:37.000000000 +0200
+++ gai-0.5.0pre6/gai.pc.in	2003-12-17 23:25:45.126776504 +0100
@@ -7,7 +7,7 @@
 
 Name: gai
 Description: A library that simplifies development of applets
-Requires: gtk+-2.0 pangoft2 @REQ_PC@ @REQ_PC_GL@
+Requires: gtk+-2.0 gthread-2.0 pangoft2 @REQ_PC@ @REQ_PC_GL@
 Version: @GAI_MAJOR@.@GAI_MINOR@.@GAI_MICRO@
 Libs: -L${libdir} -lgai
 Cflags: -I${includedir}
--- gai-0.5.0pre6/configure.in.orig	2003-10-30 19:27:00.000000000 +0100
+++ gai-0.5.0pre6/configure.in	2003-12-17 23:26:09.332096736 +0100
@@ -99,7 +99,7 @@
     if `pkg-config --exists gtk+-2.0`  ; then
 	AC_MSG_RESULT(yes)
 	GTK_CFLAGS=`pkg-config --cflags gtk+-2.0`
-	GTK_LIBS=`pkg-config --libs gtk+-2.0`
+	GTK_LIBS=`pkg-config --libs gtk+-2.0 gthread-2.0 pangoft2`
     else
 	AC_MSG_RESULT(no)
 	AC_MSG_ERROR(gtk+ 2.0.x/2.2.x is required!)
